librz/bin/bfile.c

```diff
@@ -768,6 +771,9 @@ RZ_IPI RzList *rz_bin_file_get_strings(RzBinFile *bf, int min, int dump, int raw
 			}
 		}
 		rz_list_foreach (o->sections, iter, section) {
+			if (!section->name) {
+				continue;
+			}
```

Comment:
I always wonder how a section without a name can be created... IMO it should be section->name = "" in the worst case.

Reply:
or just name it `section.noname.<offset>`